Branch-SDK/src/main/java/io/branch/referral/validators/BranchIntegrationModel.java

```diff
@@ -63,6 +64,17 @@ private void updateDeepLinkSchemes(Context context) {
         }
     }
 
+    //Function to get value from string resource
+    public String getStringResource(String stringWithIdValue){
```

Comment:
Formatting should follow code style, i.e. white space before the open curly bracket on this line, follow the same guideline on all the other places where open brackets are used. Similarly, in the `if` statements leave a space before the bracket and before each condition and before each equals sign. E.g. `if(context!=null && resourceIdArr!=null && resourceIdArr.length==2){` -> `if ( context != null && resourceIdArr != null && resourceIdArr.length == 2 ) {`